script_test.go

```diff
@@ -1219,6 +1220,22 @@ func TestExecRunsGoHelpAndGetsUsageMessage(t *testing.T) {
 	}
 }
 
+func TestExecContextCencel(t *testing.T) {
+	t.Parallel()
+	// We can't make many cross-platform assumptions about what external
+	// commands will be available, but it seems logical that 'go' would be
+	// (though it may not be in the user's path)
+	ctx, cancel := context.WithTimeout(context.Background(), 1)
```

Comment:
```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), 1 * time.Nanosecond)
```

Because it's what you do here.

Your way is valid, but I would have preferred something clearer


```suggestion
	ctx, cancel := context.WithCancel(context.Background())
	cancel()
```

Because you are creating an invalid context.

Maybe what you want is to do something like this.

```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), 200 * time.Millisecond)
```